fossology/fossology
# test(rest): Unit test cases for REST API

## Description

Adding unit test cases for REST API.

### Changes

1. Some changes are done in `Contollers` to use Helper class to get userID and groupID.
1. Created new folder `src/www/ui_tests/api/` containing all the test cases.
1. Added test suite for REST API to `src/phpunit.xml`.

## How to test

1. Run `make test-www` or `src/vendor/bin/phpunit -csrc/phpunit.xml --testsuite="Fossology PhpUnit Test Suite"` or check the Travis log.

Partial work on #1590
